src/ewatercycle/observation/usgs.py

```diff
@@ -1,99 +1,60 @@
-import os
 from datetime import datetime
 
 import numpy as np
+import pandas as pd
 import xarray as xr
 from pyoos.collectors.usgs.usgs_rest import UsgsRest
 from pyoos.parsers.waterml import WaterML11ToPaegan
 
 
-def get_usgs_data(station_id, start_date, end_date, parameter="00060", cache_dir=None):
+def get_usgs_data(
+    station_id: str, start_date: str, end_date: str, parameter: str = "00060"
+) -> xr.Dataset:
     """Get river discharge data from the USGS REST web service.
 
     See `U.S. Geological Survey Water Services
     <https://waterservices.usgs.gov/>`_ (USGS)
 
-    Parameters
-    ----------
-    station_id : str
-        The station id to get
-    start_date : str
-        String for start date in the format: 'YYYY-MM-dd', e.g. '1980-01-01'
-    end_date : str
-        String for start date in the format: 'YYYY-MM-dd', e.g. '2018-12-31'
-    parameter : str
-        The parameter code to get, e.g. ('00060') discharge, cubic feet per second
-    cache_dir : str
-        Directory where files retrieved from the web service are cached.
-        If set to None then USGS_DATA_HOME env var will be used as cache directory.
-
-    Examples
-    --------
+    Args:
+        station_id: The station id to get
+        start_date: String for start date in the format: 'YYYY-MM-dd', e.g. '1980-01-01'
+        end_date: String for start date in the format: 'YYYY-MM-dd', e.g. '2018-12-31'
+        parameter: The parameter code to get, e.g. ('00060') discharge, cubic feet per second
+
+    Returns:
+        Xarray dataset with the streamflow data
+        with unit and other metadata in the variable and global attributes.
+
+    Examples:
```

Comment:
I am unable to reproduce this example, the values for the time coordinate are different:

![image](https://github.com/eWaterCycle/ewatercycle/assets/12114825/e8f75437-3a14-44de-815e-3b585c21e954)

They seem to be object dtype instead of datetime64 dtype. The individual data points _are_ of type datetime64 though.

Reply:
Works OK for the grdc data loading.

Reply:
seems pandas, xarray and numpy treat timezones differently. I added a conversion to get rid of the timezone.